src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -9,9 +9,11 @@
     <BuildActions Condition="'$(BuildOS)' == 'windows'">$(BuildActions) -NativeToolsOnMachine</BuildActions>
     <!-- On non-windows, we don't have the JDK available and don't need to build those projects anyway because they are not consumed downstream. -->
     <BuildActions Condition="'$(BuildOS)' != 'windows'">$(BuildActions) $(FlagParameterPrefix)no-build-java</BuildActions>
-    <!-- Enable building installers on Windows. We can't build installers on Linux until https://github.com/dotnet/aspnetcore/pull/58612 as been merged and flows -->
+    <!-- Enable building installers on Windows and Linux. ASP.NET Core doesn't produce installers on Mac -->
     <BuildActions Condition="'$(BuildOS)' == 'windows'">$(BuildActions) $(FlagParameterPrefix)BuildInstallers</BuildActions>
-    <!-- <BuildActions Condition="'$(BuildOS)' != 'windows'">$(BuildActions) $(FlagParameterPrefix)build-installers</BuildActions> -->
+    <BuildActions Condition="'$(BuildOS)' == 'linux'">$(BuildActions) $(FlagParameterPrefix)build-installers</BuildActions>
+    <!-- In a source-only build, we don't pass -all, so we need to explicitly opt-in to managed components here. -->
+    <BuildActions Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(BuildOS)' == 'linux'">$(FlagParameterPrefix)build-managed</BuildActions>
```

Comment:
Interesting. How did this work before?

Reply:
When none of the `--buildX` arguments or `-all` are passed, `--build-managed` is the default. So by adding `--build-installers`, we effectively removed the implicit `--build-managed`. That's why it was only failing in source-build.

Reply:
Ah yes, makes sense. Thanks.